create-hpc-cluster/azuredeploy.json

```diff
@@ -158,377 +158,97 @@
     }
   },
   "variables": {
-    "apiVersion": "2015-05-01-preview",
-    "location": "[resourceGroup().location]",
     "virtualNetworkName": "[concat(parameters('clusterName'), 'VNet')]",
     "storageAccountNamePrefix": "[toLower(concat('hpc', uniqueString(resourceGroup().id, parameters('clusterName'))))]",
```

Comment:
@singhkay please note clusterName is an input for function uniqueString, but not for function concat. The output for function uniqueString is 13 characters, so the value of storageAccountNamePrefix is always with 16 characters.